notebooks/05_Kubeflow_Pipeline/05_04_Pipeline_SageMaker.ipynb

```diff
@@ -40,6 +40,15 @@
     "> Note: you use us-east-1, please use command `!aws s3 mb s3://$S3_BUCKET --region $AWS_REGION --endpoint-url https://s3.us-east-1.amazonaws.com`"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install sagemaker"
```

Comment:
Can you put all the required library here? Like `kfp` below.

Reply:
ok Fixed.